twine/commands/upload.py

```diff
@@ -30,6 +34,26 @@
 def skip_upload(
     response: requests.Response, skip_existing: bool, package: package_file.PackageFile
 ) -> bool:
+    """Skip uploading a package.
+
+    Return Boolean type according to the status code that responded by the repository or
+    the argument passed by the user when trying to upload the package(s).
+
+    If ``skip_existing`` is set to ``True``, then return ``False``.
+    If status code 400, 403, 409 is responded by the repository, return ``True``.
+
+    :param requests.Response response:
+        Get the response from the repository.
+    :param bool skip_existing:
+        Specify whether twine should continue uploading files if one
+        of them already exists. This primarily supports PyPI. Other
+        package indexes may not be supported.
+    :param package_file.PackageFile package:
```

Comment:
Sphinx doesn't use relative imports in the package to generate links, we need to use the fullied qualified name `twine.package_file.PackageFile`

Reply:
`twine.package_file.PackageFile` doesn't work too. 

Error given by Sphinx:
```
WARNING: py:class reference target not found: twine.package_file.PackageFile
```

Reply:
I guess we need to update `conf.py`?

Reply:
@meowmeowmeowcat Again, I'd like to spend some time with this, before you do more work on it. Apologies for the lag; I'm optimistic that I'll have time for it in the next few days.

Reply:
Thanks @bhrutledge. Then I'll stop committing to this PR first.